string msgType
int64 timestamp
string taskId
string pathId


